.pre-commit-config.yaml

```diff
@@ -77,6 +77,11 @@ repos:
       - id: isort
         args: ['--sp','setup.cfg']
         exclude: ".*(.fits|.fts|.fit|.txt|tca.*|extern.*|.rst|.md|cm/__init__.py|sunpy/extern|docs/conf.py)$"
+      - id: isort
```

Comment:
Why a second isort one?

Reply:
So the exclude list can be distinct.

Reply:
Do we want to enforce this? Could there not be a case were we would want warnings in future?